Makefile

```diff
@@ -2515,3 +2521,30 @@ certmanager_cleanup:
 	oc delete -n ${OPERATOR_NAMESPACE} csv --all --ignore-not-found=true
 	oc delete -n ${NAMESPACE} installplan --all --ignore-not-found=true
 	oc delete -n cert-manager deployment --all
+
+##@ REDIS
+.PHONY: redis_deploy_prep
+redis_deploy_prep: export KIND=Redis
+redis_deploy_prep: export NAME=redis
+redis_deploy_prep: export IMAGE=${REDIS_DEPL_IMG}
+redis_deploy_prep: export REPO=${INFRA_REPO}
+redis_deploy_prep: export BRANCH=${INFRA_BRANCH}
+redis_deploy_prep: export HASH=${INFRA_COMMIT_HASH}
+redis_deploy_prep: redis_deploy_cleanup ## prepares the CR to install the service based on the service sample file REDIS
+	$(eval $(call vars,$@,infra))
+	mkdir -p ${OPERATOR_BASE_DIR} ${OPERATOR_DIR} ${DEPLOY_DIR}
+	bash scripts/clone-operator-repo.sh
+	cp ${REDIS_CR} ${DEPLOY_DIR}
+	bash scripts/gen-service-kustomize.sh
+
+.PHONY: redis_deploy
+redis_deploy: input redis_deploy_prep ## installs the service instance using kustomize. Runs prep step in advance. Set INFRA_REPO and INFRA_BRANCH to deploy from a custom repo.
+	$(eval $(call vars,$@,infra))
+	make wait
+	bash scripts/operator-deploy-resources.sh
+
+.PHONY: redis_deploy_cleanup
+redis_deploy_cleanup: namespace ## cleans up the service instance, Does not affect the operator.
+	$(eval $(call vars,$@,infra))
+	oc kustomize ${DEPLOY_DIR} | oc delete --ignore-not-found=true -f -
```

Comment:
I wonder if we should just do an oc delete -f ${DEPLOY_DIR}/${REDIS_CR} here and remove it instead of wiping out the entire path. I'm just thinking of things like memcached that is also using this path.

Reply:
Actually won't really work because it'll be missing the values inserted by the kustomize command like the namespace.